lib/mixins/validatable-http-message.js

```diff
@@ -428,13 +429,16 @@ but body is not a parseable JSON:\n${error.message}\
     if (!contentType) {
       return false;
     }
-
-    const parsed = mediaTyper.parse(`${contentType}`);
-    const isJson =
-      (parsed.type === 'application' && parsed.subtype === 'json') ||
-      parsed.suffix === 'json';
-
-    return isJson;
+    try {
+      const { type } = contentTypeModule.parse(`${contentType}`);
```

Comment:
The only thing that troubles me is too many things sharing what seems the same namespace.
Perhaps, `contentType` argument is not really a proper content type? Or a `rawContentType`..?

Reply:
Yeah, you're right basically I just hotfixed it. I'll try to make it nicer :)

Reply:
I renamed the variables in 519965b